registrations/models.py

```diff
@@ -345,6 +351,28 @@ def signups_count(self):
     def attending_signups(self):
         return self.signups.filter(attendee_status=SignUp.AttendeeStatus.ATTENDING)
 
+    @transaction.atomic
+    def anonymize(self):
+        # Allow to anonymize signup only once
+        if self.anonymization_time is None:
+            if contact_person := getattr(self, "contact_person", None):
+                contact_person.anonymize()
+
+            if protected_data := getattr(self, "protected_data", None):
+                protected_data.extra_info = anonymize_replacement
+                protected_data.save()
```

Comment:
```suggestion
                protected_data.anonymize()
```

Reply:
Thanks, missed that one. Changed as suggested